run/rclone_archive/archive-clips.sh

```diff
@@ -4,15 +4,51 @@ log "Moving clips to rclone archive..."
 
 source /root/.teslaCamRcloneConfig
 
-FILE_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
-
-rclone --config /root/.config/rclone/rclone.conf move "$CAM_MOUNT"/TeslaCam/SavedClips "$drive:$path" --create-empty-src-dirs --delete-empty-src-dirs >> "$LOG_FILE" 2>&1 || echo ""
-
-FILES_REMAINING=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
-NUM_FILES_MOVED=$((FILE_COUNT-FILES_REMAINING))
-
-log "Moved $NUM_FILES_MOVED file(s)."
-/root/bin/send-push-message "TeslaUSB:" "Moved $NUM_FILES_MOVED dashcam file(s)."
-
+NUM_FILES=0
+NUM_FILES_MOVED=0
+NUM_FILES_IN_DIR=0
+
+function connectionmonitor {
```

Comment:
next person to copy this code somewhere gets to put it into a separate file... 😈